public/js/selectors.js

```diff
@@ -46,8 +46,7 @@ function getSelectedFrame(state: AppState) {
 }
 
 function getSourceMapURL(state: AppState, source: any) {
-  const tab = getSelectedTab(state);
-  return tab.get("url") + "/" + source.sourceMapURL;
+  return source.url + "/" + source.sourceMapURL;
```

Comment:
Needed to tweak this also because the selected tab does not exist in the Firefox context, but I think the sourceMapURL should be resolved relative to the source. This should also be a smarter `join(source.url, source.sourceMapURL)`.

I can move this out of the PR, just wanted to show everything that we need to work on.